indexer/editable_map_object.hpp

```diff
@@ -95,8 +95,8 @@ class EditableMapObject : public MapObject
 
 private:
   string m_houseNumber;
-  string m_street;
-  vector<string> m_nearbyStreets;
+  pair<string /* Default name */, string /* Localized name */> m_street;
```

Comment:
+1

> On Apr 8, 2016, at 15:28, Dmitry Yunitsky <notifications@github.com> wrote:
> 
> In indexer/editable_map_object.hpp <https://github.com/mapsme/omim/pull/2850#discussion_r59017205>:
> 
> > @@ -95,8 +95,8 @@ class EditableMapObject : public MapObject
> >  
> >  private:
> >    string m_houseNumber;
> > -  string m_street;
> > -  vector<string> m_nearbyStreets;
> > +  pair<string /* Default name */, string /* Localized name */> m_street;
> Если затайпдефить - и не придётся комментарий дублировать.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2850/files/0ad17160bff832a1d3538b656f12f37df22310a9#r59017205>